scripts/project-setup-scripts/google-project-viewer.yaml

```diff
@@ -0,0 +1,26 @@
+title: "google_project_viewer"
+description: "viewer permissions on a google-project (for example can-compute users)"
+stage: "GA"
+includedPermissions:
+# Add permissions for the roles we need:
+##roles/bigquery.jobUser (updated 2/10/2021)
+- bigquery.jobs.create
+- resourcemanager.projects.get
+- resourcemanager.projects.list
+##roles/logging.logWriter (updated 2/10/2021)
+- logging.logEntries.create
+##roles/monitoring.metricWriter (updated 2/10/2021)
+- monitoring.metricDescriptors.create
+- monitoring.metricDescriptors.get
+- monitoring.metricDescriptors.list
+- monitoring.monitoredResourceDescriptors.get
+- monitoring.monitoredResourceDescriptors.list
+- monitoring.timeSeries.create
+##RequesterPays custom role (updated 2/10/2021)
+- serviceusage.services.use
+##roles/bigquery.readSessionUser (updated 2/25/2021)
+- bigquery.readsessions.create
+- bigquery.readsessions.getData
+- bigquery.readsessions.update
+- resourcemanager.projects.get
+- resourcemanager.projects.list
```

Comment:
There are some duplicates in here.  Might not matter functionally, but for readability, I would prefer to not have them.